### PR TITLE
Unify headings titles and breadcrumbs for Core 3.0.0 beta 0 - closes #478

### LINKED ISSUE #478: Unify headings, titles and breadcrumbs for Core 3.0.0-Beta 0
- Same page title as the navigation title of the page for every page
- Update navigations to not use bold navigation links
reference/configuration reference: change title to be only "configuration"
- add banner images for Core (like in the README files on GitHub)
- Use imperative form in page titles, e.g.: Config file handling --> Handle config files
- On the start page of every version, change the H0 title to `Introduction`, remove the subtitle `Introduction` if present
- Use sentence case in titles (small letters except first letter of first word, names, abbreviations,...)
- Change titles to remove redundancy: 
  - `Setup/Binary installation` to `Setup/Binary`
  - `Setup/Docker installation` to `Setup/Docker`
  ...
  - `Update/Binary update` to `Update/Binary`
  ... and so on

### PR DESCRIPTION
**Pull request by chris529**
closes #478

### CODE REVIEW

File: modules/ROOT/pages/interact-with-network.adoc [hunk 1]
```diff
@@ -13,7 +13,7 @@ Mona Bärenfänger <mona@lightcurve.io>
 :url_explorer_testnet: https://testnet-explorer.lisk.io
 :url_explorer_betanet: https://betanet-explorer.lisk.io
 
-:url_api: reference/api.adoc
+:url_api: reference/lisk-elements/api.adoc
```
Comment (Tschakki) [inline modules/ROOT/pages/interact-with-network.adoc]: Why change this?

File: modules/ROOT/pages/management/commander.adoc [hunk 2]
```diff
@@ -1,6 +1,6 @@
 = Commander commands
 Mona Bärenfänger <mona@lightcurve.io>
-:description: The Lisk Core Commander Administration describes all relevant commands to manage Lisk Core through the Command-line with Lisk Commander.
+:description: The Lisk Core commander administration describes all relevant commands to manage Lisk Core through the command-line with Lisk Commander.
```
Comment (Tschakki) [inline modules/ROOT/pages/management/commander.adoc]:
```suggestion
:description: The Lisk Core Commander administration describes all relevant commands to manage Lisk Core through the command-line with Lisk Commander.
```

File: modules/ROOT/pages/migration.adoc [hunk 3]
```diff
@@ -85,7 +85,7 @@ rm -f lisk_bridge.sh <2>
 wget https://downloads.lisk.io/lisk/<network>/lisk_bridge.sh <3>
 ----
 
-<1> Change to lisk user.
+<1> Change to Lisk user.
```
Comment (Tschakki) [inline modules/ROOT/pages/migration.adoc]:
```suggestion
<1> Change to `lisk` user.
```

File: modules/ROOT/pages/reference/lisk-elements/packages/api-client/transactions.adoc [hunk 4]
```diff
@@ -0,0 +1,83 @@
+= Transactions
```
Comment (Tschakki) [inline modules/ROOT/pages/reference/lisk-elements/packages/api-client/transactions.adoc]: This complete page belongs to SDK docs, please remove it here

File: modules/ROOT/pages/update/source.adoc [hunk 5]
```diff
@@ -1,6 +1,6 @@
-= Source update
+= Source
 Mona Bärenfänger <mona@lightcurve.io>
-:description: The Lisk Core Source Upgrade page describes how to upgrade Lisk Core to the latest version from source.
+:description: The Lisk Core Source upgrade page describes how to upgrade Lisk Core to the latest version from source.
```
Comment (Tschakki) [inline modules/ROOT/pages/update/source.adoc]:
```suggestion
:description: The Lisk Core Source update page describes how to update Lisk Core to the latest version from source.
```

File: modules/ROOT/pages/update/source.adoc [hunk 6]
```diff
@@ -12,7 +12,7 @@ Mona Bärenfänger <mona@lightcurve.io>
 
 :url_enable_forging: management/forging.adoc
 
-This section details how to upgrade Lisk Core from source.
+This section describes how to upgrade Lisk Core from source.
```
Comment (Tschakki) [inline modules/ROOT/pages/update/source.adoc]:
```suggestion
This section describes how to update Lisk Core from source.
```

**Review by Tschakki — CHANGES_REQUESTED** [review]

File: modules/ROOT/pages/interact-with-network.adoc (hunk 1 above)
Comment (chris529) [reply modules/ROOT/pages/interact-with-network.adoc]: Ah, yes...good point..leave it as it was originally 👍

File: modules/ROOT/pages/reference/lisk-elements/packages/api-client/transactions.adoc (hunk 4 above)
Comment (chris529) [reply modules/ROOT/pages/reference/lisk-elements/packages/api-client/transactions.adoc]: Ah yes...please check, as it should now be done :-)

File: modules/ROOT/pages/interact-with-network.adoc (hunk 1 above)
Comment (Tschakki) [reply modules/ROOT/pages/interact-with-network.adoc]: Did you push the change? :)

**Review by Tschakki — CHANGES_REQUESTED** [review]

File: modules/ROOT/pages/interact-with-network.adoc (hunk 1 above)
Comment (chris529) [reply modules/ROOT/pages/interact-with-network.adoc]: yes

Comment (Tschakki) [reply modules/ROOT/pages/interact-with-network.adoc]: I don't think so. The link gets still updated. You wanted to revert it to its original state.

**Review by Tschakki — CHANGES_REQUESTED** [review]

**Review by Tschakki — APPROVED** [review]
Thank you!

**Pull request merged**